Makefile_test.common

```diff
@@ -0,0 +1,37 @@
+define test_tpl =
+$(1): $(1)/ok
+	@$$($(1)_verify); \
+	RETVAL=$$$$? ; \
+	if [ $$$$RETVAL -eq 0 ]; then \
+		echo "Test $(1) PASSED"; \
+		touch $$<; \
+		true; \
+	else \
+		echo "Test $(1) FAILED"; \
+		false; \
+	fi
+
```

Comment:
This needs sufficient documentation so that future developers understand how to use this.

Reply:
Added